tools/manifest/item.py

```diff
@@ -205,7 +205,7 @@ def to_json(self):
         if self.jsshell:
             rv[-1]["jsshell"] = True
         if self.script_metadata:
-            rv[-1]["script_metadata"] = self.script_metadata
+            rv[-1]["script_metadata"] = [(ensure_str(k), ensure_str(v)) for (k,v) in self.script_metadata]
```

Comment:
I'm still worried by the `ensure_str` here; it implies that there's still some places where we're storing different types in Python 2 and 3.

Reply:
Sure, forgot about that, I'll take a look

Reply:
Actually not, we're storing binary data in both py2 & py3 (because the file is opened in binary mode) and converting it here to `str`. Would you prefer a `decode("utf-8")` instead?